lib/puppet/type/scheduled_task.rb

```diff
@@ -92,32 +96,33 @@ def insync?(current)
       A trigger can contain the following keys:
 
       * For all triggers:
-          * `schedule` **(Required)** --- What kind of trigger this is.
-            Valid values are `daily`, `weekly`, `monthly`, or `once`. Each kind
-            of trigger is configured with a different set of keys; see the
-            sections below. (`once` triggers only need a start time/date.)
+          * `schedule` **(Required)** --- The schedule type. Valid values are
+            `daily`, `weekly`, `monthly`, or `once`.
           * `start_time` **(Required)** --- The time of day when the trigger should
             first become active. Several time formats will work, but we
             suggest 24-hour time formatted as HH:MM.
           * `start_date` ---  The date when the trigger should first become active.
             Defaults to the current date. You should format dates as YYYY-MM-DD,
             although other date formats may work. (Under the hood, this uses `Date.parse`.)
-      * For `daily` triggers:
```

Comment:
These were wrapped in code spans on purpose. The reason was to reinforce the connection between these groups of keys and whichever allowed value you fed to `schedule`, thus making it easier to read. Likewise with the extra text you deleted above in the `schedule` key. 

Is there a good reason to make these relationships less transparent?